pptx/package.py

```diff
@@ -74,7 +74,7 @@ def next_image_partname(self, ext):
         def first_available_image_idx():
             image_idxs = sorted([
                 part.partname.idx for part in self.iter_parts()
-                if part.partname.startswith('/ppt/media/image')
+                if part.partname.startswith('/ppt/media/image') and part.partname.idx is not None
```

Comment:
Oh, interesting. I think I get it now. The code assumes that all image partnames will be of the form .../image(n).xml, like .../image9.xml, which is usually the case. However it would appear that if there is one and only one image in the file, that PowerPoint names it .../image.xml, which returns `None` for partname.idx.

Can you provide a sample .pptx file that raises this error? You could just commit it as the next commit in this pull request and that would do the trick. Or any other way is fine. I'll just need to take a look at the innards (like `unzip -l pres.pptx) to inspect the part naming. and maybe have a look around for other oddities.

Was the presentation created using PowerPoint? Or perhaps some other package like LibreOffice?